src/components/entity/ha-statistic-picker.ts

```diff
@@ -99,6 +65,56 @@ export class HaStatisticPicker extends LitElement {
 
   private _init = false;
 
+  private _rowRenderer: ComboBoxLitRenderer<{
+    id: string;
+    name: string;
+    state?: HassEntity;
+  }> = (item) => html`<style>
+      paper-icon-item {
+        padding: 0;
+        margin: -8px;
+      }
+      #content {
+        display: flex;
+        align-items: center;
+      }
+      ha-svg-icon {
+        padding-left: 2px;
+        color: var(--secondary-text-color);
+      }
+      :host(:not([selected])) ha-svg-icon {
+        display: none;
+      }
+      :host([selected]) paper-icon-item {
+        margin-left: 0;
+      }
+      a {
+        color: var(--primary-color);
+      }
+    </style>
+    <ha-svg-icon .path=${mdiCheck}></ha-svg-icon>
+    <paper-icon-item>
+      <state-badge slot="item-icon" .stateObj=${item.state}></state-badge>
+      <paper-item-body two-line="">
+        ${item.name}
+        <span secondary
+          >${item.id === "" || item.id === "__missing"
+            ? html`<a
+                target="_blank"
+                rel="noopener noreferer"
+                href="${documentationUrl(
+                  this.hass,
+                  "/integrations/statistics/"
```

Comment:
```suggestion
                  "/more-info/energy/statistics"
```

(i'm opening PR to add this)